src/Details.js

```diff
@@ -1,14 +1,41 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
+import movies from "./Gallery-get";
+import Movie from './Movie';
 
 
-const Details = () => (
-    <div>
-        Hello. This will be the details page for each Movie &amp; TV Show:) <br />
-        <Link to="/" exact> Go back to the Home page. </Link>
-    </div >
+class Details extends Component {
+
+    state = {
+        message: "Hello! This will be the details page for each Movie & TV Show!:)"
+    }
+
+    changeState = () => {
+        this.setState({ message: "Comming soon!!!!" });
+    }
+
+    render() {
+        return (
+            <div>
+                {/* 
```

Comment:
Please avoid committing commented code, it's proved to be a source of zombie code in the long term, as people tends to leave things there, 'just in case' :)

Reply:
Thank you, I deleted the comments.